.circleci/config.yml

```diff
@@ -0,0 +1,17 @@
+version: 2.0
+jobs:
+  build:
+    docker:
+      - image: membrane/bionic-membrane:latest
+    environment:
+      MIX_ENV: test
+      PKG_CONFIG_PATH: '/root/app/ext/'
+    working_directory: '~/app'
+
+    steps:
+      - checkout
+      - run: update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-7 100
+      - run: apt-get install -qq libmp3lame-dev
```

Comment:
Maybe this could be baked into the image?

Reply:
It doesn't make build much slower